addon/bootstrap.js

```diff
@@ -17,12 +15,11 @@ const BASERESOURCE = "perception-study";
 
 XPCOMUtils.defineLazyModuleGetter(this, "studyUtils",
   `resource://${BASERESOURCE}/StudyUtils.jsm`);
-XPCOMUtils.defineLazyModuleGetter(this, "Feature",
+XPCOMUtils.defineLazyModuleGetter(this, "feature",
```

Comment:
I am not sure what exactly we are going to export here, the class or an instance :)